docs/diagrams/SearchSequenceDiagram.puml

```diff
@@ -0,0 +1,80 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":GradPadParser" as GradPadParser LOGIC_COLOR
+participant ":SearchCommandParser" as SearchCommandParser LOGIC_COLOR
+participant ":ParserUtil" as ParserUtil LOGIC_COLOR
+participant "d:SearchCommand" as SearchCommand LOGIC_COLOR
```

Comment:
think you have to update the "d" to "s" cause its reflecting on some of the blocks HAHAH

Reply:
Yep got it